### ISSUE TITLE
Hello, I'm confused that this program has no validation set.

### ISSUE THREAD

**Issue by mk123qwe** [DongfeiJi/Cellular-Segmentation#1]
Hello, I'm confused that this program has no validation set.

**Comment by DongfeiJi** [comment]
This program is created by google，it only contains  an  example of training data(all the data may be 100GB). Thus, it has  no validation set. Especially, if u want to learn, u can refer to https://github.com/google/ffn/

**Comment by mk123qwe** [comment]
> This program is created by google，it only contains an example of training data(all the data may be 100GB). Thus, it has no validation set. Especially, if u want to learn, u can refer to https://github.com/google/ffn/

Thank you. During FFN training, there is no validation set in https://github.com/google/ffn/.
Do you think the data size limit?

**Comment by DongfeiJi** [comment]
I think this program is mainly used for 3D reconstruction end to end, as it is not a stable work for segmentation. The entire dataset is also supplied by google but there is still no validation dataset, u can use gs to download it. MeanWhile, u can also refer to this repo, https://github.com/google/neuroglancer. It uses the entire data to gain the training result, and visualizes it in 3D.

**Comment by DongfeiJi** [comment]
yeap,it limits the data size.it is only an example, the result of train only contains one part of the flood-filling.

**Comment by DongfeiJi** [comment]
> Hello, I'm confused that this program has no validation set.

**Comment by DongfeiJi** [comment]
> > This program is created by google，it only contains an example of training data(all the data may be 100GB). Thus, it has no validation set. Especially, if u want to learn, u can refer to https://github.com/google/ffn/
> 
> Thank you. During FFN training, there is no validation set in https://github.com/google/ffn/.
> Do you think the data size limit?

yes, this repo is only an example,the training result is only a part of flood-filling.
